[2025-04-09T07:13:10.364+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-09T07:13:10.412+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_approval_dag.fetch_data manual__2025-04-09T07:13:03.832797+00:00 [queued]>
[2025-04-09T07:13:10.432+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_approval_dag.fetch_data manual__2025-04-09T07:13:03.832797+00:00 [queued]>
[2025-04-09T07:13:10.433+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-09T07:13:10.462+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_data> on 2025-04-09 07:13:03.832797+00:00
[2025-04-09T07:13:10.476+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=119) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-09T07:13:10.479+0000] {standard_task_runner.py:72} INFO - Started process 122 to run task
[2025-04-09T07:13:10.481+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_approval_dag', 'fetch_data', 'manual__2025-04-09T07:13:03.832797+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/data_approval_dag.py', '--cfg-path', '/tmp/tmperhgai9b']
[2025-04-09T07:13:10.484+0000] {standard_task_runner.py:105} INFO - Job 3: Subtask fetch_data
[2025-04-09T07:13:11.030+0000] {task_command.py:467} INFO - Running <TaskInstance: data_approval_dag.fetch_data manual__2025-04-09T07:13:03.832797+00:00 [running]> on host 3f9380c0f990
[2025-04-09T07:13:11.214+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_approval_dag' AIRFLOW_CTX_TASK_ID='fetch_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T07:13:03.832797+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T07:13:03.832797+00:00'
[2025-04-09T07:13:11.215+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-09T07:13:11.247+0000] {logging_mixin.py:190} INFO - fetch_data execute..
[2025-04-09T07:13:11.247+0000] {logging_mixin.py:190} INFO - Files in directory: ['data.xlsx']
[2025-04-09T07:13:11.247+0000] {logging_mixin.py:190} INFO - data.xlsx
[2025-04-09T07:13:11.605+0000] {logging_mixin.py:190} INFO -    ID     Name  Age Department     Contact            Email
0   1    Alice   30         HR  12345678.0  test1@gmail.com
1   2      Bob   25         IT  98765432.0  test2@gmail.com
2   3  Charlie   35    Finance         NaN  test3@gmail.com
[2025-04-09T07:13:11.605+0000] {python.py:240} INFO - Done. Returned value was:    ID     Name  Age Department     Contact            Email
0   1    Alice   30         HR  12345678.0  test1@gmail.com
1   2      Bob   25         IT  98765432.0  test2@gmail.com
2   3  Charlie   35    Finance         NaN  test3@gmail.com
[2025-04-09T07:13:11.766+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-09T07:13:11.767+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_approval_dag, task_id=fetch_data, run_id=manual__2025-04-09T07:13:03.832797+00:00, execution_date=20250409T071303, start_date=20250409T071310, end_date=20250409T071311
[2025-04-09T07:13:11.842+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-09T07:13:11.885+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-09T07:13:11.889+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
